UNRELEASED.md

```diff
@@ -1 +1,5 @@
 # Unreleased
+
+**Updated:**
+- react-native-bpk-component-touchable-overlay
+  - Updated to support custom number for the `borderRadius` field to now allow `sm`, `lg`, `pill` or `30`
```

Comment:
I think we can be clearer to end users about what the change actually does and how to use it - what do you reckon of something like this?

```suggestion
    - `borderRadius` prop now accepts numbers in addition to the existing `sm`, `lg` and `pill` values. E.g. `<BpkTouchableOverlay borderRadius={30} ... />`. 
```